Lectures/Lecture2_GitforCollaboration.md

```diff
@@ -29,10 +29,14 @@ The (physics) research community is the main community that should be considered
 + Readme, templates, community rules, licensing and credit
 
 ## Evaluation
-The students will go through lessons 6-13 of the [git tutorial](https://git.science.uu.nl/os4p-2023/git-tutorial) and will be asked to complete an exercise to demonstrate their understanding of the material.
+During the four hours of tutorial, the students will go through the fourteen lessons of the [git tutorial](https://os4p.gitbook.io/git-tutorial/) and will be asked to complete an exercise to demonstrate their understanding of the material.
```

Comment:
for people who have no experience with bash, some of the instructions in the notebook can be still quite confusing. For example, in lesson 1-setting up, I am not sure I understand myself the purpose of setting up the global username and the options in each command-line. 
The tutorial sometimes skips explaining why a command is necessary.